cl/simple_pages/templates/components.html

```diff
@@ -1,21 +1,276 @@
-<h2 class="scroll-m-20 text-3xl font-semibold tracking-tight first:mt-0">H2 Title Component</h2>
-<h3 class="scroll-m-20 text-2xl font-semibold tracking-tight">H3 Title Component</h3>
+    <h4 class="mt-3">Slot</h4>
+    <div class="bg-greyscale-100 rounded-xl p-4 my-3">
+      <div class="flex flex-col md:flex-row border-b last:border-none border-greyscale-200 max-w-full">
+        <span class="p-2.5 max-md:pb-0 md:w-60 md:min-w-60 max-md:font-medium">Default</span>
+        <span class="p-2.5 max-md:max-w-full [overflow-wrap:anywhere]">Text that will be displayed in the code block.</span>
+      </div>
+    </div>
+
+    <c-expansion-panel title="Code">
+      <c-code>
+&lt;c-code&gt;
+    Sample code here.
+&lt;/c-code&gt;
+      </c-code>
+    </c-expansion-panel>
+  </section>
+
+  <section class="max-w-full w-full border-t-2 border-greyscale-200" x-intersect="show" id="two-column-list">
+    <h2 class="mt-6 mb-3">Two column list</h2>
+    <h4 class="mt-3">Demo</h4>
+    <c-two-column-list
+      class="my-4 p-6 border border-greyscale-500 rounded-xl bg-greyscale-50 text-greyscale-600"
+      :list="[
+          {'short_name': 'First item'},
+          {'short_name': 'Second item'},
+          {'short_name': 'Third item'},
+          {'short_name': 'Fourth item'},
+      ]"
+    ></c-two-column-list>
+
+    <h4>Props</h4>
+    <div class="bg-greyscale-100 rounded-xl p-4 my-3">
+      <div class="flex flex-col md:flex-row border-b last:border-none border-greyscale-200 max-w-full">
+        <span class="p-2.5 max-md:pb-0 md:w-60 md:min-w-60 max-md:font-medium"><code>list</code></span>
+        <span class="p-2.5 max-md:max-w-full [overflow-wrap:anywhere]">List of strings that will be displayed in two even columns.</span>
+      </div>
+    </div>
+
+    <h4 class="mt-3">Slot</h4>
+    <p class="mb-3">No slot</p>
+
+    <c-expansion-panel title="Code">
+      <c-code>
+&lt;c-two-column-list
+  class="p-6 border border-greyscale-500 rounded-xl bg-greyscale-50 text-greyscale-600"
+  :list="[
+      {'short_name': 'First item'},
+      {'short_name': 'Second item'},
+      {'short_name': 'Third item'},
+      {'short_name': 'Fourth item'},
+  ]"
+&gt;&lt;/c-two-column-list>
+      </c-code>
+    </c-expansion-panel>
+  </section>
+
+  <section class="max-w-full w-full border-t-2 border-greyscale-200" x-intersect.margin.-100px="show" id="eyebrow">
+    <h2 class="mt-6 mb-3">Eyebrow</h2>
+    <h4 class="mt-3">Demo</h4>
+    <c-eyebrow class="my-4">CourtListener</c-eyebrow>
+
+    <h4 class="mt-3">Props</h4>
+    <p>No props</p>
+    <h4 class="mt-3">Slot</h4>
+    <div class="bg-greyscale-100 rounded-xl p-4 my-3">
+      <div class="flex flex-col md:flex-row border-b last:border-none border-greyscale-200 max-w-full">
+        <span class="p-2.5 max-md:pb-0 md:w-60 md:min-w-60 max-md:font-medium">Default</span>
+        <span class="p-2.5 max-md:max-w-full [overflow-wrap:anywhere]">Text to be appended.</span>
```

Comment:
The text is no longer appended within the eyebrow. let's update this description